### ISSUE TITLE
Incorporate AnimBot into your animation workflow

### ISSUE THREAD

**Issue by FeralTurtle** [FeralTurtle/Untitled_04_V2#7]
Do this at least when you’ve consistently been animating for a long time and see points in your workflow that can be improved/streamlined using AnimBot. Right now you’re barely holding onto the basics.

**Comment by FeralTurtle** [comment]
https://www.youtube.com/watch?v=YLHUS-yvMTs&t=149s (AnimBot tutorial by Skitty Animates)
https://www.youtube.com/watch?v=G49wexNmQ-Q (AnimBot tutorial by Sir Wade Neistadt)
Both explicitly advise in the first two minutes of their videos to learn how to learn to do various animation tasks without AnimBot since if you get hired into a studio and you don't have access to it, you should still be able to work in Maya.
There is one caveat that Sir Wade mentioned and that is if there's something insanely technical that it's simply practical to use a tool like AnimBot for it then go ahead but leave it to your judgement.
Basically think of AnimBot as a library of tools that act as a framework for speeding up certain tasks in Maya.
As of now, I think I'll hold off purchasing it until I'm much more serious in animation, basically animating everyday for awhile, with the free trial there if I do want to try some few things in the meantime.
